### PR TITLE
Donate to open emr

### PR DESCRIPTION
**Pull request by ruthkonyn**
Fixes #

#### Short description of what this resolves:
tiny improvement to layout. remove referrence to $string_form_title that was causing php warning. improve wording of the close button when filling in or changing the form

### CODE REVIEW

**Comment by ruthkonyn** [comment]
i am struggling with this: the conflicts that must be resolved in the message above are the changes that this PR wishes to make to the code - i don't know how to proceed.  I created a pull request, but one problem may be that the branch in my fork is well behind the openemr master.

**Comment by stephenwaite** [comment]
usually you'd create a new branch after updating master, here's the [wiki guide](https://www.open-emr.org/wiki/index.php/Using_Git_with_OpenEMR)

you can prob do this easily by putting these commands in a little script if your remotes are set up:
```
git checkout master
git fetch upstream
git pull upstream master
git push origin master
```
then checkout a new branch
`git checkout -b gad7updates`
then copy in your new files, add them, commit them and push

File: interface/forms/gad7/gad7.inc.php [hunk 1]
```diff
@@ -14,6 +14,8 @@
 
  require_once("../../globals.php");
  require_once("$srcdir/api.inc");
+ require_once("$srcdir/patient.inc");
+ /* require_once("$srcdir/globals.inc.php"); */
```
Comment (stephenwaite) [inline interface/forms/gad7/gad7.inc.php:18]: good to remove any commented code

File: interface/forms/gad7/view.php [hunk 2]
```diff
@@ -16,11 +16,13 @@
 
 use OpenEMR\Common\Csrf\CsrfUtils;    // security module
 use OpenEMR\Core\Header;
-use Mpdf\Mpdf;  /* used to generate a pdf of the form */
+/* se Mpdf\Mpdf; /* to generate pdf files*/
```
Comment (stephenwaite) [inline interface/forms/gad7/view.php:19]: ok to remove

Comment (ruthkonyn) [reply interface/forms/gad7/view.php:19]: yes please, thanks

Comment (stephenwaite) [reply interface/forms/gad7/view.php:19]: sorry, meant please go ahead and remove for us :)
also one lonely styling error
Screenshot from 2022-04-20 14-33-16

**Comment by ruthkonyn** [comment]
i apologise for the failed checks above, i'm not entirely sure how they got added to the pr - i was following suggestions in the openemr wiki document 'git for dummies' - i was trying to rebase my local git version of OpenEMR to get it up to date, and then get my github version (at ruthkonyn/openemr) up to date with a 'git push origin +donate_to_Openemr' where origin is my git repo (ruthkonyn/openemr) - i hadn't intended for the push to go all the way to the openemr repo, even of the gad7 code. 

the gad7 code that it's in the PR does include the fixes i'm trying to make, but it also has some code i'm working on, which shouldn't have an adverse affect on the form thanks to the commented out call at the end of the file report.php

**Comment by ruthkonyn** [comment]
Hi @stephenwaite.   the   PR   passed all it's checks, but for some reason merging is blocked. I'm not sure how to proceed now to get the changes incorporated? thanks

File: interface/forms/gad7/report.php [hunk 3]
```diff
@@ -12,19 +13,32 @@
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
+/* these are included in gad7.inc.php
 require_once(dirname(__FILE__) . '/../../globals.php');
 require_once($GLOBALS["srcdir"] . "/api.inc");
+*/
```
Comment (stephenwaite) [inline interface/forms/gad7/report.php:19]: would remove commented code

File: interface/forms/gad7/report.php [hunk 4]
```diff
@@ -12,19 +13,32 @@
...
 
 function gad7_report($pid, $encounter, $cols, $id)
 {
-    $count = 0;
-    $gad7_total = 0;
-    $value = 0;
+    global $str_test, $str_nervous,$gad7_total, $pdf_as_string, $str_values,$str_difficulty_values, $data, $exp, $file_path,$file_name, $str_generate_pdf;
 
-    $str_values = [0 => xl('Not at all') . ' (0)',1 => xl('Several days') . ' (1)',2 => xl('More than half of days') . ' (2)',3 => xl('Nearly every day') . ' (3)'];
+    $genpdf_file_path = '/interface/forms/gad7/report-as-pdf.php';
```
Comment (stephenwaite) [inline interface/forms/gad7/report.php:37]: `$genpdf_file_path = __DIR__ . '/report-as-pdf.php';`

**Comment by stephenwaite** [comment]
some minor stuff @ruthkonyn , only admins can merge and will wait for @bradymiller to dbl check and bring in if ready, thanks for the fixes

**Pull request closed without merging**